### PR TITLE
Data access edits

### PR DESCRIPTION
**Pull request by mikala-nsidc**
Made some updates to data access intro and part 1.

### CODE REVIEW

File: book/tutorials/data_access/data_access_1_intro.ipynb [hunk 1]
```diff
@@ -35,17 +54,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 1,
```
Comment (scottyhq) [inline book/tutorials/data_access/data_access_1_intro.ipynb:57]: Need to 'restart kernel and clear all outputs'

**Review by asteiker — APPROVED** [review]
I ran into an error in data_access_2 here:

```
auth = Auth()

try: 
    auth.login(strategy='netrc')
except:
    auth.login(strategy='interactive')```

But I believe @mikala-nsidc is addressing this (the assumption should be that most peopel have their .netrc set up before this tutorial). Also I asked a question in the education team slack on whether to point to /tmp or ./tmp - my feeling is that we should point to the latter to avoid confusion with a hidden folder not showing up in the JupyterLab directory but not sure what the best practice is across each tutorial for creating new data folders.

**Comment by scottyhq** [comment]
> I ran into an error in data_access_2 here

@asteiker thanks for the review! What error specifically? 

> whether to point to /tmp or ./tmp - my feeling is that we should point to the latter to avoid confusion with a hidden folder not showing up in the JupyterLab directory but not sure what the best practice is across each tutorial for creating new data folders. 

It's true people won't see /tmp from the jupyterlab file browser, but you can `ls /tmp` in a terminal. Or actually people can create a softlink in the home directory `ln -s /tmp /home/jovyan/tmplink` in order to see things in the file browser. The reason for putting data in /tmp is actually that anything under `/home/jovyan` persists in the jupyterhub, so we quickly start accumulating unnecessary costs

**Comment by asteiker** [comment]
@scottyhq Cool, I knew there had to be a reason! And that's a very valid reason to keep costs down! Thanks for the explanation and helpful tips on how to create a softlink - I can see that being useful this week. 

I'll copy the error with the try / except code here to document in one place, though I know we have a separate slack thread going as well:

```Expects .netrc in /home/jovyan
[Errno 2] No such file or directory: '/home/jovyan/.netrc'
```

**Comment by mikala-nsidc** [comment]
I'm done pushing edits!  This can be merged to main anytime.  Do I do that?  Or @scottyhq  or @JessicaS11 ?

**Comment by scottyhq** [comment]
> I'm done pushing edits! This can be merged to main anytime. Do I do that? Or @scottyhq or @JessicaS11 ? 

I believe we set things up so that anybody on the "planning-2022" github org team can merge with at least 1 other approving review. I just messed up the tests with a change to our workflows, so will push a couple changes to fix, then merge!

**Pull request merged**